backend/app/main.py

```diff
@@ -166,39 +230,27 @@ async def post_listings(
         return Field500ErrorResponse(error="Internal Server Error. Please try again later.")
 
 @app.post(
-    '/sign-up',
-    response_model=None,
-    responses={
-        '201': {'model': SignUpPostResponse},
-        '400': {'model': SignUpPostResponse1},
-        '409': {'model': SignUpPostResponse2},
-    },
+    '/sign-up', response_model=None, responses={'201': {'model': SignUpPostResponse}}
```

Comment:
Might have been a merge conflict but is there a reason why the generated code didn't list the appropriate responses here.